docs/source/modules/tutorial.rst

```diff
@@ -0,0 +1,653 @@
+8. ``brain-score.web``: website front and back end.
+9. ``Brainio_contrib`` (archived): used in the past to contribute stimuli and datasets
+   (now part of ``brainio_collection``).
+
+Which repos will I use?
+-----------------------
+When we get to the install guide, I will show you exactly how to
+clone/fork repos for your own project in the easiest way possible.
+But for now, you will mainly only need the ``sample-model-submission`` repo.
+
+How do I get a Brain-Score for my model?
+----------------------------------------
+
+Brain-Score currently has two “ways" to score your model:
+
+1. *On your machine*:  your model is scored on one or more benchmarks (a benchmark is a
+   standard to compare against, like Majaj-Hong’s 2015 paper on IT results (which you
+   can read more about here: https://brain-score.readthedocs.io/en/latest/modules/benchmarks.html). This is useful if you would like to test and see if your
+   model is ready to be submitted and run against all benchmarks, or if you want a quick
+   and dirty way to score on a single benchmark, like V4 or IT. This is mainly used to
+   test and/or optimize your model before submitting to the main site.
+2. *Remote*: your model is run on Brain-Score’s website, and contains scores against all the
+   local benchmarks, plus many others (we avoid sharing this code with users to
+   avoid overfitting). The (hierarchical) mean of all these benchmark scores is the
+   Brain-Score itself. This has more overhead to get started, but is worth it, as you
+   can set up pull requests to our main code package (i.e, suggest changes) if you find
+   errors and have the packages/dependencies automatically installed.
+
+Why do you recommend installing and submitting the way outlined in this guide? In other words, why should I do it your way?
+------------------------------------------------------------------------------
+
+A reasonable question, and it is always good to be skeptical. The short answer
+is that using an IDE like Pycharm and virtual environments
+drastically cuts the error rate for install down, as well as makes the whole
+process of installing dependencies easier. Using a venv also helps with headaches
+caused by clashes between Anaconda and PIP, and Pycharm
+(or another IDE like VScode) takes care of that for you.
+
+Do I have to read/do this entire tutorial to submit a model?
+------------------------------------------------------------
+
+No - You can skip to the Quickstart section below, if you do not
+wish to read/do this entire tutorial, but we recommend it in
+order to get the most out of Brain-Score and avoid errors that might pop up along the way.
+
+
+
+Quickstart
+==========
+In this section, we will provide a quick and easy way to get Brain-Score
+up and running. This is mainly for those who do not have the time to read
+or do the whole tutorial, or for those who just want to go ahead and submit
+a model quickly; however, we do officially recommend that you read and do the
```

Comment:
"we do officially recommend that you read and do the entire tutorial, as it will help you in the future." -> "we recommend referring back to this tutorial, especially if you encounter errors."